crates/event-watcher-traits/src/evm/event_watcher.rs

```diff
@@ -21,6 +22,10 @@ use super::*;
 pub type EthersClient =
     providers::Provider<providers::RetryClient<providers::Http>>;
 
+/// Ethereum TimeLag client using Ethers, that includes a retry strategy.
+pub type EthersTimeLagClient =
+    TimeLag<Arc<providers::Provider<providers::RetryClient<providers::Http>>>>;
```

Comment:
Move the TimeLag above the RetryClient, not over the Arc.
```suggestion
    Arc<TimeLag<providers::Provider<providers::RetryClient<providers::Http>>>>;
```